android/src/com/mapswithme/maps/dialog/EditTextDialogFragment.java

```diff
@@ -46,8 +46,8 @@ public static void show(String title, String initialText, String hint, String po
     final Bundle args = new Bundle();
     args.putString(EXTRA_TITLE, title);
     args.putString(EXTRA_INITIAL, initialText);
-    args.putString(EXTRA_POSITIVE_BUTTON, positiveBtn.toUpperCase());
-    args.putString(EXTRA_NEGATIVE_BUTTON, negativeBtn.toUpperCase());
+    args.putString(EXTRA_POSITIVE_BUTTON, positiveBtn);
```

Comment:
Дак не переводит, это я и добавил недавно как раз с этой целью. Предлагаю добавить @NonNull в сигнатуру и поправить места вызова.

> On May 10, 2016, at 1:13 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/maps/dialog/EditTextDialogFragment.java <https://github.com/mapsme/omim/pull/3139#discussion_r62645409>:
> 
> > @@ -46,8 +46,8 @@ public static void show(String title, String initialText, String hint, String po
> >      final Bundle args = new Bundle();
> >      args.putString(EXTRA_TITLE, title);
> >      args.putString(EXTRA_INITIAL, initialText);
> > -    args.putString(EXTRA_POSITIVE_BUTTON, positiveBtn.toUpperCase());
> > -    args.putString(EXTRA_NEGATIVE_BUTTON, negativeBtn.toUpperCase());
> > +    args.putString(EXTRA_POSITIVE_BUTTON, positiveBtn);
> Система сама их в верхний регистр приведёт. Крэш был из-за того, что сюда null может передаваться.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3139/files/b4de72a720950e839c43f23e200251c0af423fcb#r62645409>